docs/userguide/declarative_config.rst

```diff
@@ -192,28 +192,28 @@ obsoletes                                          list-comma
 Options
 -------
 
-=======================  ===================================  =============== =========
+=======================  ===================================  =============== ====================
 Key                      Type                                 Minimum Version Notes
-=======================  ===================================  =============== =========
+=======================  ===================================  =============== ====================
 zip_safe                 bool
-setup_requires           list-semi                            36.7.0
-install_requires         list-semi
-extras_require           section                                              [#opt-2]_
+setup_requires           file:, list-semi                     36.7.0          [#opt-5]_
```

Comment:
Question: would it make more sense to restrict this change to `install_requires` and `extras_requires`?

`tests_require` is deprecated, so I feel like we should not touch it.

`setup_requires` is largely obsoleted by PEP 517.
In very specific use cases people might still reach for it inside `setup.py`, but not in the context of `setup.cfg`.

I understand that adding support for the `file:` directive for `setup_requires` and `tests_require` basically come for free, but I still think we should refrain from doing that.